modules/core/shared/src/main/scala/coursier/maven/PomParser.scala

```diff
@@ -95,7 +95,12 @@ object PomParser {
 
     var description = ""
     var url = ""
-    val licenses = Nil // TODO
+    val licenses = Nil // Left in for binary compat
```

Comment:
See my comment above, this can be removed, and corresponding MiMa filters can be added in `project/Mima.scala`.
```suggestion
```